NPM uploads seem not to be working

Hi USWDS,

I tried to follow the instructions on `master` to install via NPM, but it would seem that the NPM registry is returns the following: 
`npm install --save uswds --registry https://registry.npmjs.org/`

```
npm ERR! node v0.10.24
npm ERR! npm  v3.7.2
npm ERR! code E404

npm ERR! 404 Registry returned 404 for GET on https://registry.npmjs.org/uswds
npm ERR! 404
npm ERR! 404  'uswds' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

and additionally:

https://registry.npmjs.org/uswds returns simply `{}`

--- a/README.md
+++ b/README.md
@@ -47,7 +47,7 @@ Draft Web Design Standards.
 
 ```shell
 cd path/to/project-using-npm
-npm install --save uswds
+npm install --save git+https://git@github.com/18F/web-design-standards.git#18f-pages-staging --registry https://registry.npmjs.org/
 ```
 
 This will add the Draft Web Design Standards as a dependency for your project and

--- a/README.md
+++ b/README.md
@@ -50,6 +50,12 @@ cd path/to/project-using-npm
 npm install --save git+https://git@github.com/18F/web-design-standards.git#18f-pages-staging --registry https://registry.npmjs.org/
 ```
 
+> __Please note__ that installing the `uswds` package via Github will use the
+development version of the Draft Web Design Standards and is not tied to a
+stable release. Installation via Github will also be deprecated once this package
+is published to `npm`. The library may also have a different structure so some
+file paths may be different.
+
 This will add the Draft Web Design Standards as a dependency for your project and
 save it to your `package.json` file. The package will be installed in `node_modules`
 under `uswds`.

